### PR TITLE
Ep 1 Finished (together with script and umi_ftr)

### PR DESCRIPTION
**Pull request by KnuckleBuckle**
Finished Umineko Episode 1 (together with coding and translation of the script folder and umi_ftr.txt file)

### CODE REVIEW

**Comment by vit9696** [comment]
Just for a heads up, this is on my list, and I will try to review it soon. Thanks for the effort!

File: script/it/caches.txt [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+		getcursorpos %x,%y
+		amsp 5,%x,%y-33
+		print 1
+		getscreenshot 1920,1080
+		savescreenshot "caches\kakera_memory8.png"
+		d_continue
+		wait_on_d -1
+	textoff
+	lsp s0_3,kakera_memory10_src,0,0
+	if localisation != "it" jumpf
```
Comment (vit9696) [inline script/it/caches.txt:171]: Why was that added? I do not see it in English.

**Comment by vit9696** [comment]
This looks good to me besides the line I quoted. Was it tested locally?

**Comment by vit9696** [comment]
Please also add script uploading to the CI:
https://github.com/umineko-project/umineko-scripting/blob/master/.github/workflows/main.yml

You will need `last_it` file with ep1 for Italian.

File: script/it/caches.txt (hunk 1 above)
Comment (KnuckleBuckle) [reply script/it/caches.txt:171]: Yeah, I'll immediately cut line 171. Most probably an oversight.

**Comment by KnuckleBuckle** [comment]
> This looks good to me besides the line I quoted. Was it tested locally?

Yes, it was, and we found no problems in executing the onscripter-ru.
